### ISSUE TITLE
Update gorilla emoji to full gorilla instead of just face

### ISSUE THREAD

**Issue by brandonhaslegs** [twitter/twemoji#478]
The gorilla emoji doesn't match [any other platform](https://emojipedia.org/gorilla/). It should be redrawn to be a full-sized gorilla instead of just a face.

**Comment by Crissov** [comment]
Everybody else had 🦍 as a face originally as well, but meanwhile converged on Apple's design choice, unfortunately.

Unicode should adopt ZWJ sequences to distinguish full-body and facial animal emojis (and their should be gender variants in some cases #164).

**Comment by brandonhaslegs** [comment]
Whether the choice was unfortunate or not is not exactly the point, although I totally agree that users should be able to choose to display a head or a body (or in some cases choose the gender) for animal emoji. I bring this issue up purely from a user experience perspective. When a user taps on a full-sized gorilla emoji on _any_ platform, Twemoji displays a gorilla head. That's unexpected and feels like a mistake.

**Comment by brandonhaslegs** [comment]
I would even argue that in this case displaying only the head of the gorilla is incorrect, because the emoji is called "Gorilla," not "Gorilla Face".

Examples of this:
`Cow Face 🐮 (U+1F42E)` vs `Cow 🐄 (U+1F404)`
`Pig Face 🐷 (U+ 1F437)` vs `Pig 🐖 (U+ 1F416)`

**Comment by Crissov** [comment]
No, those a legacy distinctions. Showing either, head or body, is fine.

**Comment by brandonhaslegs** [comment]
🤔 How can they be legacy but also still be the current and official name of the character? Can you link me to something that explains which emojis are legacy? On unicode.org, which I assume is up to date, these characters are still called "Cow," "Cow Face," "Pig," and "Pig Face."

https://unicode.org/emoji/charts/emoji-list.html#1f42e
https://unicode.org/emoji/charts/emoji-list.html#1f437
https://unicode.org/emoji/charts/emoji-list.html#1f416
https://unicode.org/emoji/charts/emoji-list.html#1f404

**Comment by brandonhaslegs** [comment]
Even if they wait to draw the full body gorilla when Unicode adopts ZWJ sequences for animals, the fact remains that the full body gorilla needs to be drawn, so the work needs to be done regardless. Might as well do it now, switch to a full body gorilla to conform with all other platforms, and when Unicode adopts ZWJ sequences for animals, then Twemoji will already have both ready to go!

**Comment by Crissov** [comment]
What I meant by ”legacy distinction“ is that new animal emojis will not come in pairs of two codepoints for both, head and body. This newer policy does not recommend one glyph design over the other, however. 

I think it was U+1F992 🦒 with the character name _Giraffe Face_ that sparked the discussion at the time, which resulted in the emoji name becoming just _Giraffe_.
